src/routes/chrome/tests/websocket.spec.ts

```diff
@@ -509,4 +509,53 @@ describe('Chrome WebSocket API', function () {
 
     await browser.disconnect();
   });
```

Comment:
Just a heads up that I don't think any of these tests will hit that change since they're purely on the playwright side. You'll need to add `/playwright` to the path in your connect calls with playwright in order to test fully